### PR TITLE
use urllib3.ProxyManager if proxy environment is set

### PR DESCRIPTION
**Pull request by cwt**
Instead of showing a warning message:

`UserWarning: httplib2shim does not use proxy_info, specify a custom pool instead.`

function `_default_make_pool` should return urllib3.ProxyManager.

### CODE REVIEW

**Comment by cwt** [comment]
I signed it!

2017-04-13 18:14 GMT+07:00 googlebot <notifications@github.com>:

File: httplib2shim/__init__.py [hunk 1]
```diff
@@ -114,7 +125,13 @@ def _conn_request(self, conn, request_uri, method, body, headers):
             scheme, host, conn.port, request_uri)
 
         decode = True if method != 'HEAD' else False
-
+        if isinstance(self.pool, urllib3.ProxyManager):
```
Comment (theacodes) [inline httplib2shim/__init__.py]: will the pool not do this automatically when we call `pool.request`?

Comment (cwt) [reply httplib2shim/__init__.py]: no, urllib3 did parse the proxy_url, but it didn't create or update proxy_headers. however, I already simplify the code by create proxy_headers beforehand, and remove that if-clause in my last commit.

**Comment by theacodes** [comment]
Travis is failing for unrelated reasons, merging.

**Comment by cwt** [comment]
Thank you for merging. Could you also create a new version in PyPI?

**Comment by theacodes** [comment]
@cwt I can, but I want to get the tests passing again. In the meantime you can point pip directly at this git repo.

**Comment by theacodes** [comment]
@cwt published 0.0.2.

**Pull request merged**